docs/sts-oidc-cloudfront.json.tpl

```diff
@@ -0,0 +1,91 @@
+{
+    "DistributionConfig": {
+        "CallerReference": "${CLUSTER_NAME}",
+        "Aliases": {
+            "Quantity": 0
+        },
+        "Origins": {
+            "Quantity": 1,
+            "Items": [
+            {
+                "Id": "${OIDC_BUCKET_NAME}.s3.${CLUSTER_REGION}.amazonaws.com",
+                "DomainName": "${OIDC_BUCKET_NAME}.s3.${CLUSTER_REGION}.amazonaws.com",
+                "OriginPath": "",
+                "CustomHeaders": {
+                    "Quantity": 0
+                },
+                "S3OriginConfig": {
+                    "OriginAccessIdentity": "origin-access-identity/cloudfront/${OAI_CLODUFRONT_ID}"
+                },
+                "ConnectionAttempts": 3,
+                "ConnectionTimeout": 10,
+                "OriginShield": {
+                "Enabled": false
+                }
+            }
+            ]
+        },
+        "DefaultCacheBehavior": {
+            "TargetOriginId": "${OIDC_BUCKET_NAME}.s3.${CLUSTER_REGION}.amazonaws.com",
+            "TrustedSigners": {
+                "Enabled": false,
+                "Quantity": 0
+            },
+            "TrustedKeyGroups": {
+                "Enabled": false,
+                "Quantity": 0
+            },
+            "ViewerProtocolPolicy": "https-only",
+            "AllowedMethods": {
+                "Quantity": 2,
+                "Items": [
+                    "HEAD",
+                    "GET"
+                ],
+                "CachedMethods": {
+                    "Quantity": 2,
+                    "Items": [
+                        "HEAD",
+                        "GET"
+                    ]
+                }
+            },
+            "SmoothStreaming": false,
+            "Compress": false,
+            "LambdaFunctionAssociations": {
+                "Quantity": 0
+            },
+            "FunctionAssociations": {
+                "Quantity": 0
+            },
+            "FieldLevelEncryptionId": "",
+            "CachePolicyId": "4135ea2d-6df8-44a3-9df3-4b5a84be39ad"
+        },
+        "CacheBehaviors": {
+            "Quantity": 0
+        },
+        "CustomErrorResponses": {
+            "Quantity": 0
+        },
+        "Comment": "${CLUSTER_NAME}",
+        "Logging": {
+            "Enabled": false,
+            "IncludeCookies": false,
+            "Bucket": "",
+            "Prefix": ""
+        },
+        "PriceClass": "PriceClass_All",
+        "Enabled": true,
+        "ViewerCertificate": {
+            "CloudFrontDefaultCertificate": true
+        }
+    },
+    "Tags": {
+        "Items": [
+            {
+                "Key": "Name",
+                "Value": "${CLUSTER_NAME}"
+            }
+        ]
+    }
```

Comment:
No at this document (create flow). But it can be useful in two ways:
- Identify the CloudFront reference on the Console
- The destroy flow can use that tag to lookup (I believe we can add more standards tags here, wdyt?)

Reply:
Makes sense. Lets keep these for now. We can add other standard tags with ccoctl process.